telesto/object_detection/app.py

```diff
@@ -0,0 +1,85 @@
+import json
+from typing import List, Dict
+
+import falcon
+import numpy as np
+
+from telesto.config import config
+from telesto.resources.simple import RootResource, DocsResource
+from telesto.object_detection.model import DummyObjectDetectionModel
+from telesto.object_detection import DetectionObject
+from telesto.utils import convert_base64_images_to_arrays
+
+PNG_RGB_GRAY8_BASE64_FORMAT = {
+    "type": "png",
+    "palette": "RGB24 or GRAY8",
+    "encoding": "base64",
+    "max_size": "5120",
+}
+
+PREDICT_ENDPOINT_DOCS = {
+    "path": "/",
+    "method": "POST",
+    "name": "Predict endpoint",
+    "request_body": {
+        "images": [
+            {
+                "content": "<str>"
+            }
+        ]
+    },
+    "image_format": {
+        **PNG_RGB_GRAY8_BASE64_FORMAT,
+    },
+    "response_body": {
+        "predictions": [
+            {
+                "objects": [
+                    {
+                        "x": "<int>",
+                        "y": "<int>",
+                        "w": "<int>",
+                        "h": "<int>",
+                    }
+                ]
+            }
+        ]
+    },
```

Comment:
Is the `class` missing here? The json has a `classes` field right below, so I guess every object should have this field as well.

Reply:
Yes, we will need support of classes. However, it'll be added separately.